drivers/flash/Kconfig

```diff
@@ -36,6 +36,63 @@ config FLASH_PAGE_LAYOUT
 	help
 	  Enables API for retrieving the layout of flash memory pages.
 
+config SPI_FLASH_MX25
```

Comment:
create a Kconfig.mx25 and put all this content into it. Or then the root flash Kconfig will grow and grow etc...

Of course first option becomes menuconfig and not only config.